versions.yaml

```diff
@@ -152,14 +152,14 @@ assets:
 
   kernel:
     description: "Linux kernel optimised for virtual machines"
-    url: "https://cdn.kernel.org/pub/linux/kernel/v4.x/"
+    url: "https://cdn.kernel.org/pub/linux/kernel/v5.x/"
     uscan-url: >-
       https://mirrors.edge.kernel.org/pub/linux/kernel/v5.x/linux-(5\.4\..+)\.tar\.gz
     version: "v5.10.25"
 
   kernel-experimental:
     description: "Linux kernel with virtio-fs support"
-    url: "https://cdn.kernel.org/pub/linux/kernel/v4.x/"
+    url: "https://cdn.kernel.org/pub/linux/kernel/v5.x/"
```

Comment:
It would be nice that the scripts uses this URL to pull the kernel tarball. Just in case you want to work in a future PR as well.

Reply:
Good idea. I just looked into that, and I this script lets you specify any version. It parses out the major version to build the right URL. So it looks like this doesn't use the versions.yaml URL for an appropriate reason.